hw/ip/adc_ctrl/data/adc_ctrl.hjson

```diff
@@ -162,29 +165,29 @@
         resval: "0",
         fields: [
 //          { bits: "1:0",
-//	    name: "min_ext",
-//	    desc: "2-bit extension; RO 0",
-//	  }
+//          name: "min_ext",
+//          desc: "2-bit extension; RO 0",
+//        }
```

Comment:
Why keep these comments?Also below..

Reply:
i asked the same question before... and the response was those are reserved fields we might use later. 
There's no real use for them right now though.